### ISSUE TITLE
The testing results

### ISSUE THREAD

**Issue by ZhimaoPeng** [CanPeng123/FSCIL_ALICE#1]
When I run the training code i.e. "main_base.py", I got the following results:
acquiring class-wise feature prototype from training data ...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 235/235 [00:02<00:00, 102.11it/s]
acquiring feature prototype for testing data ...
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:00<00:00, 73.33it/s]
Session 0 | NCM accuracy: 0.8031666666666667 
Saving the model at the last epoch.
It seems normal.
But when I run the testing code i.e. "main_inc_ncm.py",, I got the following results:
=> creating model 'resnet18'
=> loading checkpoint '/home/ubuntu/code/open-set/FSCIL_ALICE/exp/CIFAR100_base-data-2-fusion_cosFace_batch-128_epoch-100_lr-1e-2_milestones-60-70_seed-1_s-30_m-4e-1/1/trial1_session0_best.pth'
=> loaded pre-trained model '/home/ubuntu/code/open-set/FSCIL_ALICE/exp/CIFAR100_base-data-2-fusion_cosFace_batch-128_epoch-100_lr-1e-2_milestones-60-70_seed-1_s-30_m-4e-1/1/trial1_session0_best.pth'
session: 0
number of images: 30000
~~~~~~~~ training dataset ~~~~~~~~
Files already downloaded and verified
---- CIFAR100 Base Transform ---
~~~~~~~~ testing dataset ~~~~~~~~
---- CIFAR100 Testing Transform ---
length of the trainset: 30000
----------------------------- calculate and store average class-wise feature embedding -----------------------------
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 235/235 [00:02<00:00, 93.17it/s]
----------------------------- do interference -----------------------------
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:00<00:00, 79.58it/s]
TEST, total average accuracy=0.1545
TEST, task-wise correct prediction: [927, 0, 0, 0, 0, 0, 0, 0, 0]
TEST, task-wise number of images: [6000, 0, 0, 0, 0, 0, 0, 0, 0]
TEST, task-wise accuracy: [0.1545, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
This looks weird. Did I do something wrong?  Can you provide a testing configuration? thank you!

**Comment by CanPeng123** [comment]
The testing configurations have been uploaded.
